test/allocations/src/overflow_from_zero.d

```diff
@@ -0,0 +1,6 @@
+void main()
+{
+    void[] buffer;
+    buffer.length = 1;
+    buffer.length = size_t.max;
```

Comment:
This file seems to have exactly the same content as `overflow_from_existing.d`. Is that on purpose?